### PR TITLE
update w & b error

### PR DESCRIPTION
**Pull request by Ricky-WL**
#13 PR for update w&b error.
Add 3-tuple regparam for bias, one-way interactions and pair-wise interactions

### CODE REVIEW

File: src/main/scala/org/apache/spark/mllib/classification/FFMWithAdag.scala [hunk 1]
```diff
@@ -34,12 +34,12 @@ import scala.util.Random
   *            interactions, respectively.
   * @param n_iters number of iterations
   * @param eta step size to be used for each iteration
-  * @param lambda regularization for pairwise interactions
+    * @param regParam A (Double, Double, Double) 3-Tuple stands for regularization params of bias, one-way interactions and pairwise interactions
```
Comment (VinceShieh) [inline src/main/scala/org/apache/spark/mllib/classification/FFMWithAdag.scala]: indentation

File: src/main/scala/org/apache/spark/mllib/classification/FieldAwaredFactorizationMachine.scala [hunk 2]
```diff
@@ -46,7 +46,7 @@ import scala.collection.mutable.WrappedArray
   *            interactions, respectively.
   * @param n_iters number of iterations
   * @param eta step size to be used for each iteration
-  * @param lambda regularization for pairwise interations
+    * @param regParam A (Double, Double, Double) 3-Tuple stands for regularization params of bias, one-way interactions and pairwise interactions
```
Comment (VinceShieh) [inline src/main/scala/org/apache/spark/mllib/classification/FieldAwaredFactorizationMachine.scala]: indentation

File: src/main/scala/org/apache/spark/mllib/classification/FieldAwaredFactorizationMachine.scala [hunk 3]
```diff
@@ -163,7 +163,7 @@ object FFMModel extends Loader[FFMModel] {
 
     /** Model data for model import/export */
     case class Data(numFeatures: Int, numFields: Int, dim0: Boolean, dim1: Boolean, dim2: Int,
-                      n_iters: Int, eta: Double, lambda: Double, isNorm: Boolean,
+                      n_iters: Int, eta: Double, reg1: Double, reg2:Double, reg3:Double, isNorm: Boolean,
```
Comment (VinceShieh) [inline src/main/scala/org/apache/spark/mllib/classification/FieldAwaredFactorizationMachine.scala]: a space missing after reg2 and reg3

**Review by VinceShieh — COMMENTED** [review]
Neat! Just some minor coding style changes required.
Thanks for the PR!

**Comment by Ricky-WL** [comment]
Modification of new commit:
1. fix indentation and space problems
2. remove reg param of bias
Please help review it, thanks.

**Comment by VinceShieh** [comment]
Merged. Thanks!

**Pull request merged**